### PR TITLE
Feature/charges api

### PR DESCRIPTION
**Pull request by AGS4NO**
Adds the billing API charges endpoint

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1763

### CODE REVIEW

**Review by wilson-dnsimple — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This project uses [Semantic Versioning 2.0.0](http://semver.org/).
 
 ## main
 
+## 1.3.0
```
Comment (DXTimer) [inline CHANGELOG.md:7]: If we add a version then we should also append `(Unreleased)` to the version so if the change ends up in main there is no confusion if a developer does not find a corresponding release/tag.

File: src/Dnsimple/Client.php [hunk 2]
```diff
@@ -81,6 +82,10 @@ class Client
      * @var Accounts The service handling the Accounts API
      */
     public $accounts;
+    /**
+     * @var Billing The service handling the Billing API
+     */
+    public $billing;
```
Comment (DXTimer) [inline src/Dnsimple/Client.php:88]: This needs to be registered at the bottom of the file.

**Review by DXTimer — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AGS4NO) [reply CHANGELOG.md]: Moved to main section, will update version during release

File: src/Dnsimple/Client.php (hunk 2 above)
Comment (DXTimer) [reply src/Dnsimple/Client.php]: @AGS4NO this has not yet been resolved. The change needs to be made to https://github.com/dnsimple/dnsimple-php/blob/4c48d82ba25bffc475e1264a0bb48ef443149379/src/Dnsimple/Client.php#L287C16-L287C16

**Review by DXTimer — APPROVED** [review]

**Pull request merged**